channels/channel.be/vier/chn_vier.py

```diff
@@ -362,7 +366,11 @@ def create_search_result(self, result_set: dict) -> MediaItemResult:
         if item_type == "video":
             item = MediaItem(title, url, media_type=mediatype.VIDEO)
         elif item_type == "program":
-            item = FolderItem(title, url, content_type=contenttype.EPISODES)
+            if data["tracking"] and data["tracking"]["item_category"] == "Film":
+                url = url.replace(self.baseUrl, f"{self.baseUrl}/video")
+                item = MediaItem(title, url, media_type=mediatype.VIDEO)
```

Comment:
Weet niet of de category "Film" altijd werkt, maar het zal in ieder geval beter zijn dan het was! 

Zou je wel nog `mediatype.MOVIE` kunnen maken dan?

Reply:
Aangepast 😊 Ik heb een stuk of tien films bekeken en die hadden allemaal `item_category` als `Film`. Ik heb nog een andere manier gevonden, bij films is `program` altijd leeg, ik zal die ook nog even toevoegen.